.eslintrc.json

```diff
@@ -16,6 +16,7 @@
         "@typescript-eslint"
     ],
     "rules": {
+    //   "semi": ["error", "never"],
```

Comment:
Do we need this?

Reply:
This was more of a proposal for us to start not using semicolons; ill remove it